components/frontend-react/src/API/Downloads.js

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+
+const fetchDownloads = () => {
+  let token = sessionStorage.getItem('token');
+  return axios.get(
+    `${process.env.REACT_APP_API_GET_DOWNLOADS}`,
+    {headers: {'token': `${token}`}}
+  );
+}
+
+const startAllDownloads = () => {
+  return axios.get(
+    `${process.env.REACT_APP_API_START_DOWNLOADS}`,
+    {headers: {'key': '123456789'}}
+  );
+}
+
+const killAllDownloads = () => {
+  return axios.get(
+    `${process.env.REACT_APP_API_KILL_DOWNLOADS}`,
+    {headers: {'key': '123456789'}}
+  );
+}
+
+const addNewDownload = (link) => {
+  let obj = {link: link};
```

Comment:
`let reqData = {link} `

Since the Object property shorthand syntax we don't need to repeat `{link:link}` if the variable name and property name is equal.

Reply:
I will make the necessary changes